src/server/auth.ts

```diff
@@ -50,10 +50,10 @@ export const authOptions: NextAuthOptions = {
   },
   adapter: PrismaAdapter(db) as Adapter,
   providers: [
-    // DiscordProvider({
-    //   clientId: env.DISCORD_CLIENT_ID,
-    //   clientSecret: env.DISCORD_CLIENT_SECRET,
-    // }),
+    DiscordProvider({
```

Comment:
A best practice, but something I forget to do a lot is to update the env.example file so developers coming into the project have a list of each env they'll need.